src/Services/Utils/SQLUpgradeService.php

```diff
@@ -1243,4 +1250,48 @@ private function convertLayoutProperties()
             } // end group
         } // end form
     }
+
+    private function updateLayoutEditOptions($mode, $form_id, $add_option, $values)
+    {
+        $subject = explode(',', str_replace(' ', '', $values));
+
+        $sql = "SELECT `field_id`, `edit_options`, `seq` FROM `layout_options` WHERE `form_id` = ? ";
+        $result = sqlStatementNoLog($sql, array($form_id));
+
```

Comment:
should have a simple flag/mechanism to follow if anything actually gets changed.
then when call this function, if something is changed, return true
and is something is not changed, then return false
then can incorporate this return in the place where function is called to show if the call is "skipped" or not